src/core/Track.cpp

```diff
@@ -1125,25 +1133,176 @@ void TrackContentObjectView::contextMenuEvent( QContextMenuEvent * cme )
 	if( fixedTCOs() == false )
 	{
 		contextMenu.addAction( embed::getIconPixmap( "cancel" ),
-					tr( "Delete (middle mousebutton)" ),
-						this, SLOT( remove() ) );
+					tr( individualTCO
```

Comment:
Can we fix the indention of all the addAction calls? All the indention is crazy with unnecessary tabs.

Reply:
Sure, I tried to keep the indentation as it was originally, but it can be fixed. Should it be this instead?

```
    contextMenu.addAction( embed::getIconPixmap( "cancel" ),
        tr( individualTCO ? "Delete (middle mousebutton) " : "Delete selection (middle mousebutton)" ),
        [this](){ contextMenuAction( remove ); } );
```

Reply:
Second option is cleaner to me, just pull the `embed` part down as well.
```
contextMenu.addAction(
	embed,
	tr,
	[]()
);
```